### ISSUE TITLE
Easy Photography Portfolio plugin

### ISSUE THREAD

**Issue by brenoalvs** [brenoalvs/monk#250]
Let's make it compatible with Easy Photography Portfolio plugin

**Comment by pyronaur** [comment]
Cool, how can I help?

I'm using internationalized strings everywhere throughout the plugin, and CMB2 to power the admin panel.

Here is the source: https://github.com/justnorris/easy-photography-portfolio


This is what was required to make it compatible with WPML: https://github.com/justnorris/easy-photography-portfolio/blob/master/compatibility/wpml.php

**Comment by brenoalvs** [comment]
@justnorris I found 3 main issues:

- The "main portfolio page" are not filtering entries by language.
- I needed flush my permalinks to be able to access portfolio categories.
- The Monk language switcher is not showing portfolio category translations links on their pages.

Can you do more tests on your side to look for anything else?
And then I'll break this issue in some subtasks to investigate what is going on.

**Comment by pyronaur** [comment]
Yeah, the main page is using WP_Query to just query all `phort_post` posts. And I think the permalinks needed a refresh because EPP registers a new post type, so it's not really an incompatibility.

I'll try to take a closer a look at it when I can.

**Comment by brenoalvs** [comment]
We should be filtering any WP_Query by language for default.
I think that the first and third items are with us. Don't worry about them!
